serverlessworkflow/modules/ROOT/pages/cloud/operator/referencing-resource-files.adoc

```diff
@@ -0,0 +1,155 @@
+====
+
+You should have a `ConfigMap` with two data entries similar to this one:
+
+.Example of a ConfigMap containing the data for the worflow
+[source,yaml,subs="attributes+"]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: service-resources
+data:
+  workflow-service-schema.json: # data was removed to save space
+  workflow-service-openapi.json: # data was removed to save space
+----
+
+Now you can reference this `ConfigMap` to your `SonataFlow` CR:
+
+.SonataFlow CR referencing a ConfigMap resource
+[source,yaml,subs="attributes+"]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlow
+metadata:
+  name: service
+  annotations:
+    sonataflow.org/description: Hello Service!
+    sonataflow.org/version: 0.0.1
+    sonataflow.org/profile: dev
+spec:
+  resources: <1>
+    configMaps:
+      - configMap:
+          name: service-resources <2>
+        workflowPath: specs <3>
+  flow:
+    start: Service
+    dataInputSchema: specs/workflow-service-schema.json
+    functions:
+    - name: isWinner
+      operation: specs/workflow-service-openapi.json#isWinner
+      type: rest
+    states:
+    - name: Service
+      type: operation
+      actions:
+      - name: CallService
+        functionRef:
+          refName: isWinner
+    end: true
+----
+
+<1> Introduced a new attribute `.spec.resources` where you can bind the `ConfigMap` to the `SonataFlow` CR
+<2> The name of the `ConfigMap` in the same namespace
+<3> The path where we want to reference these resources
+
+Note that the `workflowPath` is `specs`. This is the path where you want to reference the files within the `ConfigMap` in the workflow definition.
+
+[IMPORTANT]
+====
+Always create your `ConfigMaps` before the `SonataFlow` since not having the resources available during startup might break the deployment.
+====
+
+Any resource you have to map to the flow definition can be added to the `SonataFlow` CR using this procedure.
+
+== Creating a Static Service Registry
+
+The `ConfigMap` containing workflow resources are not tied to a particular `SonataFlow` instance, just referenced by it. It means that the operator won't edit or delete them if the `SonataFlow` CR is updated or deleted. You have total control of the `ConfigMap` instance.
+
+That said, you can organize your `ConfigMaps` in a way that other workflows could reuse them. In other words, one `ConfigMap` can be mapped to many `SonataFlow` Custom Resources.
+
+For example, if you have many OpenAPI or xref:eventing/orchestration-of-asyncapi-based-services.adoc[AsyncAPI] specification files that your workflows can access, you can group them by domain. This way you create a static Service Registry using `ConfigMaps`. Other developers within your company can reuse the same specification files when designing workflows.
+
+The same applies for data input and output schemas, subflows definitions, and so on.
+
```

Comment:
Here is missing the types handled with Quarkus, 

- Camel Routes, 
- OpenAPI, 
- AsyncAPI, 
- Generic 

 and the default root path for workflowPath, `src/main/resources/`

Reply:
Doesn't matter the type anymore, actually. But I'll add the default mount path. 👍

Reply:
Actually, I don't think it matters for end users. This is an implementation detail, I won't add here.